Fix shortcuts in Maya 2016 + Add shortcut for selecting the closest camera

yann-lty opened the pull request:
Connected to #175
- Fix hotkeys (shorcuts) : Maya 2016 uses a new hotkey system. The one by default is read-only, so we select a writable one to assign our shortcuts or create a new one if needed.
- Added a shortcut to select the closest camera (Ctrl+F by default)
- Fixed QmlInstantCoding; just have to define MAYAMVG_QML_PATH=1 in the environment

yann-lty committed:
Update .gitignore

Ignore build* folders

yann-lty committed:
QmlInstantCoding fix

* Use MVGMainWidget.cpp file location to determine qml source folder for instant coding
* Build fix: added missing include for usleep function

yann-lty committed:
Turn "closest camera selection" into a Maya command

* Algorithm stays the same
* MVGProjectWrapper now calls this command

yann-lty committed:
Fix hotkeys in Maya >= 2016

Ensure the current hotkey set is writable (i.e. not the default one)

yann-lty committed:
Added shortcut for selecting closest camera

Binded by default on Ctrl+F

eprana commented:
LGTM

Just be careful with MGlobal::executeCommand(). 
Prefere MGlobal::executeCommandOnIdle() when possible.

fabiencastan commented:
As we don't use a path anymore, we can rename the MAYAMVG_QML_PATH into MAYAMVG_USE_QMLINSTANTCODING=1.

yann-lty committed:
Change QML Instant Coding env. var name

Use MAYAMVG_USE_QMLINSTANTCODING = 1 for in-source QML development with hot reload
